ext/standard/tests/streams/stream_socket_sendto.phpt

```diff
@@ -27,32 +27,32 @@ if (is_resource($sock)) {
 }
 ?>
 --EXPECTF--
-Notice: fwrite(): send of %i bytes failed with errno=%i Broken pipe in %s on line %i
+Notice: fwrite(): send of %i bytes failed with errno=%i Broken pipe in %s on line %d
```

Comment:
While generally changing `%i` to `%d` here makes sense, I wonder why that is not done in php/php-src. Is there, maybe, a behavioral difference between run-tests.php and PFTT2?

Reply:
I am unsure. As a note for this test in particular, I am unsure why it has continuously failed on my setup. However, I did know that this file was changed before and it currently works. Therefore, I updated it here.

Reply:
Oh, indeed the placeholders have been changed with php/php-src@53dbc69a8db8ce76db7eef4cebc1483cc2edf790. However, it shouldn't make a difference whether `%d` is used or `%i` (the latter accepts everything the former does, plus an optional +/- sign). We may consider to apply something like php/php-src@d9094d4d6455811c2bd9f23511ec0adbdb607b4f, or simply drop the test as has been done in php/php-src@84333cad674890174c47f3c21b1b1cf85a4276ac.

Reply:
Well, for me it keeps failing regardless of changing it according to the first commit. Dropping the test seems like the easier way to go. I'll probably just drop this test then if that's fine?

Reply:
Yes, dropping this test should be fine, since indeed "it doesn't seem to test anything particularly
useful".

Reply:
Sounds good. Removed.